adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java

```diff
@@ -83,4 +84,20 @@ public void extractRssiFromLoraMessage() {
         Assert.assertEquals(48.0, map.getOrDefault(LoraConstants.APP_PROPERTY_RSS, null));
     }
 
+    /**
+     * Verifies that the extracted gateways are correct.
+     */
+    @Test
+    public void extractGatewaysFromLoraMessage() {
+        final JsonObject loraMessage = LoraTestUtil.loadTestFile("actility.uplink");
+        final Map<String, Object> map = provider.extractNormalizedData(loraMessage);
+        Assert.assertTrue(map.containsKey(LoraConstants.GATEWAYS));
+
+        JsonArray expectedArrays =
+                new JsonArray("[{\"gateway_id\":\"18035559\",\"rss\":48.0,\"snr\":3.0},{\"gateway_id\":\"18035560\"," +
```

Comment:
can you please create the JSON objects in the array using `JsonObject`?

Reply:
I could have expected this :)